client/src/app/pages/tasks/tasks-page.tsx

```diff
@@ -189,6 +190,11 @@ export const TasksPage: React.FC = () => {
     columnState,
   } = tableControls;
 
+  const tooltips: Record<string, ThInfoType> = {
```

Comment:
```suggestion
  const tooltips: Record<string, ThProps["info"]> = {
```

It's easy to import `ThProps` and then reference the info prop.  Then there are no oddball imports required.